cypress/integration/premium/admin.spec.ts

```diff
@@ -1,270 +1,310 @@
 import CONSTANTS from "../../support/constants";
 
-const { GOOD_PASSWORD } = CONSTANTS;
+const { GOOD_PASSWORD, CONFIG_INTEGRATIONS_AUTOMATIONS } = CONSTANTS;
 
-const getConfig = {
```

Comment:
Moved to software.spec.ts instead of duplicated in both free/admin.spec.ts and premium/admin.spec.ts